### PR TITLE
Fixes #7644: Permissions of rudder tools are not preserved on relays

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7644

### CODE REVIEW

File: techniques/system/common/1.0/update.st [hunk 1]
```diff
@@ -320,7 +320,7 @@ bundle agent update_action
         comment           => "Check if we need to update the tools";
 
     # We copy only if we have new tools based on the timestamp
-    new_tools_available::
+    !policy_server.new_tools_available::
```
Comment (amousset) [inline techniques/system/common/1.0/update.st]: In case there is already a rudder_tools_updated file.

**Comment by ncharles** [comment]
are you sure the issue is only on 3.2 ?

**Comment by amousset** [comment]
The problem in the code is there in 3.0 (and 2.11) too, the difference is somehow in the initial copy which preserved the permissions before 3.2. If a file in Rudder tools is then changed on the server, it triggers the update.cf copy which preserves permissions, so we never saw the permission problem happen.
Retargeting to 2.11.

**Comment by amousset** [comment]
Replaced by #828.

**Pull request closed without merging**